db/seeds.rb

```diff
@@ -31,7 +31,7 @@ def load_crops
   Dir.glob("#{source_path}/crops*.csv").each do |crop_file|
     puts "Loading crops from #{crop_file}..."
```

Comment:
I'd be inclined to move this whole for-loop into `CsvImporter.import_file`, but no biggie.